presto-spi/src/main/java/com/facebook/presto/spi/function/GrpcScalarFunctionImplementation.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spi.function;
+
+public class GrpcScalarFunctionImplementation
```

Comment:
Hmm, this looks exactly the same as `ThriftScalarFunctionImplementation`. Maybe we should just rename it to `RemoteScalarFunctionImplementation` and add a `FunctionImplementationType` to denote which protocol is in use instead of adding another class?